### ISSUE TITLE
[adascale] benchmark test

### ISSUE THREAD

**Issue by min-xu-ai** [facebookresearch/fairscale#234]
It would be nice to have a benchmark test for adascale with cifar that does the full training or a few epochs just to make sure the gain value and training loss is going correctly etc. Also, it can check the overall speed with and without adascale.

**Comment by anj-s** [comment]
@min-xu-ai Is this something we should work on in the future? Would this be a good starter task?
